docs/troubleshooting.md

```diff
@@ -35,3 +36,28 @@ using Minishift.
   systemctl start virtlogd
   systemctl enable virtlogd
   ```
+
+
+## XHYVE drivers
+
+- Could not create vmnet interface, permission denied or no entitlement
+
+    The problem is xhyve driver is not able to clean up vmnet when remove a VM.
+
+    Workaround:
+        vmnet.framework seems to have decide the IP based on below files
+
+            /var/db/dhcpd_leases
+            /Library/Preferences/SystemConfiguration/com.apple.vmnet.plist
+
+    So, If you want to reset `minishift` specific IP database, then make sure you remove `minishift` entry section from `dhcpd_leases` file and reboot your system.
+
```

Comment:
When a user faces the above error he will have to do this step right? So we can drop the 'If' and say: Reset the `minishift` specific IP database and make sure you remove the `minishift` entry section from the `dhcpd_leases` file. Finally, reboot your system.